new/PRS13990ZawaldaMaryam.xml

```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13990ZawaldaMaryam" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ዘወልደ፡ ማርያም፡</title>
```

Comment:
The convention is to have the name here in transliteration

Reply:
I was not able to detect the name of Zawalda Maryam on fol. 12rb, despite my efforts, sorry (is the number correct? The folio numbers are very small and poorly visible, and do not correspond to the numbers of the images). I dealt only with the scribal hand of one of the units. In principle, it is not impossible that Zawalda Maryam of this ID is identical to `PRS12076ZawaldaMaryam`, but this can hardly be proved. I would say, the dating of Unit 1 could well be 17/18th century, the flourishing time for this Zawalda Maryam can be perhaps changed accordingly .

Reply:
I changed the title phrase to "Zawalda Māryām".

Reply:
The name occurs in the third line. What is confusing is, that the second part ማርያም is in red, while ዘወልደ is in black ink. Perhaps the entire text was copied including the name of the possessor or this Vorlage what might be a reason to date this person even earlier.
The first three lines in folio 12rb are:
ሁ፡ ለመንፈስ፡ ቅዱስ፡ እማኅፀ
ንኩ፡ ዝነፍሰ፡ ኃጥእ፡ ገብርከ፡ 
ዘወልደ፡ ማርያም። አሜን። ወ
I did not pay much attention to the paleography of the handwriting but relied on Marrassinis judgement. If I note 17th, 18th or 19th century, this is a very long time span. Do you think it is useful to put it like this or is it more appropriate to delete the date completely?
I changed it to 17th, 18th or 19th century / notBefore: 1600, notAfter: 1900 for now. I guess, I need to make this change also to the manuscript record.
My assumption is, that it is the same hand for the entire manuscript. Do you agree?

Reply:
Yes, 17-19 cent. is too long (it is like the cataoguer would say: "I do not know how to date it", I think we should avoid this). One century spam, or a bit more, as dating suggestion is usually possible on palaeographical grounds.

Reply:
I completely agree that a one-century span is more desirable, but a three-century span is still better than no dating at all or a erroneously narrow span, in cases where the cataloguer (like me) does not have the ability to date the hand more precisely.

Reply:
Palaeographic dating is cannot be precise, and no one claims/expects this, but a three century span may encompasses up to three palaeographic periods and as such does not make much sense.

Reply:
I work mostly with your summer school handout, in which period VI encompasses mid17th-mid19th c. :) In any case, should such cases come up, I will simply contact you :)

Reply:
This is a handout :)